generators/spa/templates/VueJS/common/Settings.html

```diff
@@ -1,19 +1,27 @@
 [JavaScript:{ jsname: "JQuery" }]
-[JavaScript:{ path: "https://cdn.jsdelivr.net/npm/vue/dist/vue.js"}]
+[JavaScript:{ jsname: "Knockout" }]
 [JavaScript:{ path: "~/Resources/Shared/scripts/dnn.jquery.js"}]
+[JavaScript:{ path: "~/DesktopModules/<%= moduleName %>/scripts/common.js"}]
 [JavaScript:{ path: "~/DesktopModules/<%= moduleName %>/scripts/QuickSettings.js"}]
 
-<div id="vue-settings-wrapper-[ModuleContext:ModuleId]" class="dnnForm">
-    <div id="settings-[ModuleContext:ModuleId]">
-        <div class="dnnFormMessage dnnFormInfo">
-            <div>{{message}}</div>
-            [Resx:{key:"SettingsInfo"}]
-        </div>
+<div id="QuickSettings-[ModuleContext:ModuleId]" class="dnnForm">
```

Comment:
@VAlmea Is it possible to apply any of the HTML structure/branding/design from the following resource?  https://uxguide.dnnsoftware.com/

Reply:
@VAlmea Do you think this question/suggestion will have any positive benefits?